buildSrc/src/main/groovy/com/github/okbuilds/okbuck/generator/BuckFileGenerator.groovy

```diff
@@ -133,6 +133,18 @@ final class BuckFileGenerator {
             ":${rule.name}"
         }
 
+        // SqlDelight
+        if (target.sqldelight != null) {
+            String runner = target.sqldelight.externalDeps.find { it.contains("SqlDelightBin") }
+            if (runner == null) {
+                throw new IllegalStateException("SqlDelight model module must declare the dependency: " +
+                        "`provided(group: 'com.github.okbuilds', name: 'SqlDelightBin', version: '1.0.0', ext: 'jar', classifier: '')`")
```

Comment:
we should leave version of SqlDelightBin out of the error.

Actually okbuck plugin can automatically apply this provided dependency if sqldelight plugin is enabled. so users dont need to specify explicitly unless they want to change version